src/main/java/nextstep/fp/Conditional.java

```diff
@@ -0,0 +1,9 @@
+package nextstep.fp;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+public interface Conditional {
+
+    int getCondition(int total, int num);
```

Comment:
```suggestion
    boolean getCondition(Integer number);
```

위와 같이 해야 간결하게 람다로 구현이 가능할것 같아요 🤔

Reply:
습관대로 진행하다보니 복잡한 형태로 람다를 구현하게 된 것 같습니다.
보통 람다는 최대한 간단하게 표현하는게 좋은 케이스일까요?